ComponentKit/DataSources/Common/CKComponentDataSourceAttachController.mm

```diff
@@ -102,15 +101,11 @@ - (void)_detachComponentLayoutFromView:(UIView *)view
 
 static CKComponentDataSourceAttachState *_mountComponentLayoutInView(CKComponentLayout layout,
                                                                      UIView *view,
-                                                                     CKComponentScopeRootIdentifier scopeIdentifier,
-                                                                     CKComponentBoundsAnimation boundsAnimation)
+                                                                     CKComponentScopeRootIdentifier scopeIdentifier)
 {
   CKCAssertNotNil(view, @"Impossible to mount a component layout on a nil view");
   NSSet *currentlyMountedComponents = view.ck_attachState.mountedComponents;
-  __block NSSet *newMountedComponents = nil;
-  CKComponentBoundsAnimationApply(boundsAnimation, ^{
-    newMountedComponents = CKMountComponentLayout(layout, view, currentlyMountedComponents, nil);
-  }, nil);
```

Comment:
probably this needs to be ported for our internal data source fork — we need to do this!